File: core/Dtos/OrderItemBriefDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using core.Entities.Orders;

namespace core.Dtos
{
    public class OrderItemBriefDto
    {
        public int Id {get; set;}
        public int OrderItemId { get; set; }
        public bool RequireInternalReview { get; set; }
        public int OrderId { get; set; }
        public int OrderNo {get; set;}
        public string CustomerName { get; set; }
        public string AboutEmployer {get; set;}
        public DateTime OrderDate { get; set; }
        public int CategoryId { get; set; }
        public string CategoryName { get; set; }
        public string CategoryRef {get; set;}
        public string CategoryRefAndName {get; set;}
        public int Quantity { get; set; }
        public string Status { get; set; }
        public JobDescription JobDescription {get; set;}
        public Remuneration Remuneration {get; set;}
        public bool AssessmentQDesigned {get; set;}
    }
}

File: client2/src/app/shared/dtos/admin/orderItemBriefDto.ts

export interface IOrderItemBriefDto
{
     checked: boolean;
     orderNo: number;
     orderDate: Date;
     customerName: string;
     orderItemId: number;
     requireInternalReview: boolean;
     categoryRef: string;
     categoryName: string;
     categoryRefAndName: string;
     quantity: number;
     status: string;
     assessmentQDesigned: boolean;
}

export class OrderItemBriefDto implements IOrderItemBriefDto
{
     checked: boolean = false;
     orderNo = 0;
     orderDate= new Date();
     customerName='';
     orderItemId=0;
     requireInternalReview = false;
     categoryId = 0;
     categoryRef='';
     categoryName = '';
     categoryRefAndName='';
     quantity=0;
     status='';
     assessmentQDesigned = false;
}